### ISSUE TITLE
How to cite pre-trained model

### ISSUE THREAD

**Issue by kat-kel** [CompNet/Tibert#1]
I'm impressed by the speed of the batched inference in this early release and I'd like to implement it a research project. Could you please let me know how you prefer we cite your model? (And is there an upcoming paper?)

**Comment by Aethor** [comment]
Hi, thank you for reaching out!
There likely wont be an upcoming paper for this repository specifically (but there will probably be a paper for [Renard](https://github.com/CompNet/Renard), the pipeline for which Tibert was made for). However, you can cite Tibert like so:

```bibtex
@Misc{tibert,
  author = {Amalvy, A. and Labatut, V. and Dufour, R.},
  title = {Tibert},
  year  = {2023},
  url   = {https://github.com/CompNet/Tibert},
}
```

I will add this to the README to be future proof.

**Comment by Aethor** [comment]
Side note if you're using the pip package: I made a lot of bugfixs since the last release. I just released v 0.2.0 with these, so don't hesitate to upgrade! (`pip install tibert --upgrade`)
